cgc/kmeans.py

```diff
@@ -8,6 +8,15 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_STATISTICS = [
+    (np.mean, None),
+    (np.std, None),
+    (np.percentile, {"q": 5}),
+    (np.percentile, {"q": 95}),
+    (np.max, None),
+    (np.min, None),
+]
+
 
 class KmeansResults(Results):
```

Comment:
Uhm, you mean the functions to compute the cluster statistics? Unfortunately we cannot, since all the attributes added to the `KmeansResults` object need to be serialised in the JSON output file. This is why I am adding instead the name of the functions (and not the functions themselves), together with the optional additional arguments, to the result object in line 134.

Reply:
ah okay, I did not realized it need to be serialized to JSON. Then this is good for me! Please go ahead and merge